linux/Platform.c

```diff
@@ -430,117 +431,91 @@ char* Platform_getProcessEnv(pid_t pid) {
    return env;
 }
 
-/*
- * Return the absolute path of a file given its pid&inode number
- *
- * Based on implementation of lslocks from util-linux:
- * https://sources.debian.org/src/util-linux/2.36-3/misc-utils/lslocks.c/#L162
- */
-char* Platform_getInodeFilename(pid_t pid, ino_t inode) {
```

Comment:
Please move the removal of the `Platform_getInodeFilename` implementation for the Linux platform into the other patch with the general cleanup of this; just re-order the commits accordingly in your PR. Makes it easier to review and cleans this bugfix patch.

Reply:
applied